### ISSUE TITLE
Does the expression type support audio?

### ISSUE THREAD

**Issue by Vepnar** [guiferviz/mnemocards#17]
Configuration similar to the audio settings of vocabulary type would be nice. I tried adding copy the settings with no success.

```
{
    "packages": [
        {
            "name": "english",
            "decks": [
                {
                    "id": "b698a07e-70eb-474b-82cb-f244a608b21e",
                    "name": "English",
                    "src": [
                        {
                            "type": "expression",
                            "file": "expressions.tsv",
                            "header": true,
                            "card_color": "#AA3333",
                            "card_properties": {
                                "tags": ["english", "expressions"]
                            }
                        }
                    ],
                    "audio": {
                        "lang": "en",
                        "media_dir": "./media"
                    }
                }
            ]
        }
    ]
}
```

**Comment by Otter-man** [comment]
@Vepnar
> If I download the audio myself. Can I link the audio easily with the cards?

No, so far mnemocards allow audio only for `vocabulary` and `autogenerated` types and only generated by google.
There is no way currently to add audio (even if you downloaded it beforehand) to other card types during generation. You can only add audio manually after creating a deck through anki interface.

I'll look into it though and maybe in the following month add support for user-created audio for `markdown ` type and autogeneration audio for `expression`. 

However mnemocards maintainer @guiferviz is seem to be busy with other projects so approving of pull requests might take some time. In the meantime, if I add audio support as you ask about - I'll do it in my own fork and will send you a mention.

**Comment by guiferviz** [comment]
Hi @Vepnar , as @Otter-man said there is not support for audio generation in *expression* cards.

I'll be a bit more free for the next few months, I'll design what I want to have in version 1.0 during the next few days and post it somewhere in GitHub, in case you're interested. In a nutshell the changes will be mostly a refactor so that anyone can easily create plugins (card types, like *vocabulary*, *expressions* and *cards*) and also modify the style of the cards without having to modify the library code.

Thanks again for your support @Otter-man .
